.github/workflows/main.yml

```diff
@@ -0,0 +1,65 @@
+name: CI/CD
+
+on:
+  push:
+    branches: [INV-26/deploy-to-vps]
```

Comment:
Let's keep the branches in the environment too.
branches:
      - ${{ secrets.TARGET_BRANCH }